<launch>

    <node name="passthrough_filter" pkg="my_pcl_tutorial" type="passthrough_filter"  output="screen">
        <remap from="input" to="/velodyne_points"/>

        <param name="field_name" value="x"/>
        <param name="upper_limit" value="10.0"/>
        <param name="lower_limit" value="0.0"/>

        <remap from="output" to="passthrough_points"/>
    </node>

</launch>